app/assets/stylesheets/ama_layout/layout_components/sections.scss

```diff
@@ -25,6 +25,14 @@
   }
 }
 
+.section-scroll{
```

Comment:
I know that the other class names in this file don't follow the convention, but this should be section--scroll